internals/cli/cmd_exec.go

```diff
@@ -56,6 +57,7 @@ var execDescs = map[string]string{
 	"gid":     "Group ID to run command as",
 	"group":   "Group name to run command as (group's GID must match gid if both present)",
 	"timeout": "Timeout after which to terminate command",
+	"context": "Run the command in the context of this service (overridden by -w, --env, --user, --uid, --group, --gid)",
```

Comment:
Maybe "Inherit the context (env, user/group settings, working dir) of the named service"

Also with the list: if we change anything, we'll have to either update this list, or (more likely) forget to and it will go out of sync. Better not to list things here IMO, just say "overridden by exec options" as you have in other doc comments.